helm/cfgov/Chart.yaml

```diff
@@ -29,6 +29,14 @@ dependencies:
     version: "11.6.19"
     repository: "https://charts.bitnami.com/bitnami"
     condition: postgresql.enabled
+  - name: opensearch
+    version: "2.0.0"
```

Comment:
Does the chart version line up with the OpenSearch version? Are we going to hit any incompatibilities between OpenSearch 1.x and its ES7 compatibility and the 2.0 added here?

Reply:
Chart and OpenSearch versions do line up in this case. Not an issue right now since we're just using the Opensearch APIs on the backend. Will need to be changed later.